src/RevProc.cc

```diff
@@ -1720,9 +1719,8 @@ bool RevProc::ClockTick( SST::Cycle_t currentCycle ){
                     id, HartToExec, GetActiveThreadID(), ExecPC);
     #endif
 
-    // attempt to execute the instruction as long as it is NOT
-    // the firmware jump PC
-    if( ExecPC != _PAN_FWARE_JUMP_ ){
+
+    if ( true ){
 
```

Comment:
I see the motivation to preserve the logic of the previous implementation, but I think we can safely remove this :)